src/server/install/index.md

```diff
@@ -9,6 +9,14 @@ Installation guide will help you to install your <CommunityPlatformNameLink /> o
 
 We recommend using a dedicated host machine with 8 GB of memory. The requirements for CPU and persistent storage depend largely on the frequency of project updates and the anticipated size of the data you expect to store respectively.
 
+For a typical <MainPlatformName /> enterprise level deployment for a small team (<20 users), the following specs are sufficient:
```

Comment:
see line 10 - shouldn't it be in line with the table?

Reply:
@varmar05 I need some help here trying to figure out some specs for different teams sizes on MM. Without any metrics is difficult to make some assumptions. Should we add some metrics/monitoring to our cloud instance?